mix.exs

```diff
@@ -131,6 +131,15 @@ defmodule Archethic.MixProject do
 
   defp aliases do
     [
+      compile: ["git_hooks.install", "compile"],
+      "dev.update_deps": [
```

Comment:
when this task will be called ?

Reply:
i added new aliases for updating deps, instead of doing one by one manually

Reply:
Yes but we should have a way to avoid outdated deps

Reply:
AN idea could be to let the deps outdated check in the CI but at the end and non blocking.
So outdated PR will not block a commit push, but we will see that there is outdated deps after all commands are done

Reply:
> Yes but we should have a way to avoid outdated deps

we have to do it manually, as it may include feature that break existing code.

Reply:
> AN idea could be to let the deps outdated check in the CI but at the end and non blocking. So outdated PR will not block a commit push, but we will see that there is outdated deps after all commands are done

With this approach, I'm afraid we will not look at it, unless the CI is failing. In CI's job, we just want it to pass.